react/src/lib/components/SubsurfaceViewer/layers/axes2d/axes2DLayer.ts

```diff
@@ -467,6 +506,36 @@ Axes2DLayer.defaultProps = defaultProps;
 
 //-- Local help functions. -------------------------------------------------
 
+function word2view(
+    viewMatrix: number[],
+    pos_w: vec4
+): [number, number, number] {
+    const pos_v = vec4.transformMat4(
+        vec4.create(),
+        pos_w,
+        mat4.fromValues(
+            viewMatrix[0],
+            viewMatrix[1],
+            viewMatrix[2],
+            viewMatrix[3],
+            viewMatrix[4],
+            viewMatrix[5],
+            viewMatrix[6],
+            viewMatrix[7],
+            viewMatrix[8],
+            viewMatrix[9],
+            viewMatrix[10],
+            viewMatrix[11],
+            viewMatrix[12],
+            viewMatrix[13],
+            viewMatrix[14],
+            viewMatrix[15]
+        )
+    );
+
+    return [pos_v[0], pos_v[1], pos_v[2]];
```

Comment:
```suggestion
    return pos_v.slice(0, 3);
```

Reply:
Unfortunately those suggestions gives problems with typescript. mat4.fromValues for example requires exactly 16 argument.